### ISSUE TITLE
Repo maintained?

### ISSUE THREAD

**Issue by sebj54** [NativeScript/theme#295]
Hi {N} team!

I was wondering if this repository is still maintained. `3.0.1` version is out for 9 months now but it does not appear on this repository. It appears only on [npm](https://www.npmjs.com/package/@nativescript/theme) and in app templates.

I wanted to update the npm module in my project but could not see a CHANGELOG or a release note explaining the migration process. So is it planned to migrate this repository to a package in NativeScript/NativeScript repo?

**Comment by sebj54** [comment]
For those like me who are wondering what is the difference between `2.3.4` and `3.0.1`, here are the changes:

```diff
diff -ur node_modules/@nativescript/theme ../ns-debug-stripe/node_modules/@nativescript/theme
diff -ur node_modules/@nativescript/theme/package.json ../ns-debug-stripe/node_modules/@nativescript/theme/package.json
--- node_modules/@nativescript/theme/package.json	2021-09-21 17:22:01.000000000 +0200
+++ ../ns-fresh-app/node_modules/@nativescript/theme/package.json	2021-09-22 15:14:49.000000000 +0200
@@ -1,6 +1,6 @@
 {
   "name": "@nativescript/theme",
-  "version": "2.3.4",
+  "version": "3.0.1",
   "description": "NativeScript Core Theme",
   "main": "index",
   "typings": "index.d.ts",
```

**Comment by NathanaelA** [comment]
@sebj54 - Jean, that is correct the differences between 3.01 and 2.3.4 are none.     However the differences between 3.0 and 3.0.1 (and 2.3x and 2.34) are considerable.  I am no longer working for nStudio now so I can't state anything **officially**, but at that point I was still working for nStudio and I had several issues to track down in this repo.  

Long story short:
When Nathan Walker created the 3.0 branch for NS 7 it was incompatible with NS 6 but some of the NS templates were hard coded to look for theme "~3.0.0" which of course started breaking people who were starting new NS 6 projects in that period.     Second, I had a issue that affected for  NS 5, 6 &  7.   So I fixed the issues and made 3.0 backwards compatible and then released it both as 2.3.x and 3.0.1 so that both people on "~2.3.x" and "~3.0.x" (i.e. older NS 5,6 & NS 7) users would get the exact same fixes and make sure any maintenance going forward was easier.   :)

Now as to the maintenance of this repo, I personally suspect you will see no more changes.   Igor has already started the Tailwind CSS repo which was something we talked about for a long time: https://github.com/NativeScript/tailwind and as far as I know that once completed will be the "official" supported method for theming in NS.    However, with that said you can ping Igor or Nathan on any PR's and they probably would release a new version with that PR.

**Comment by sebj54** [comment]
Hi @NathanaelA, thank you for this well detailed answer :) I understand now why there isn't a 3.x branch if it's the same code.

I'm not really a big fan of Tailwind's philosophy but I also understand that it would be very time consuming to maintain two things. Let's wait and see!
